File: 백준/Bronze/10101. 삼각형 외우기/삼각형 외우기.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        
        int A = sc.nextInt();
        int B = sc.nextInt();
        int C = sc.nextInt();
        
        if (A + B + C != 180) {
            System.out.println("Error");
        } else {
            if (A == 60 && B == 60 && C == 60) {
                System.out.println("Equilateral");
            } else if (A == B || B == C || A == C) {
                System.out.println("Isosceles");
            } else {
                System.out.println("Scalene");
            }
        }
    }
}

File: 백준/Bronze/10101. 삼각형 외우기/삼각형 외우기.cs
using System;

class Q10101
{
    static void Main(string[] args)
    {
        int A = int.Parse(Console.ReadLine());
        int B = int.Parse(Console.ReadLine());
        int C = int.Parse(Console.ReadLine());
        
        if (A + B + C != 180)
        {
            Console.WriteLine("Error");
        }
        else
        {
            if (A == B && B == C && A == C)
            {
                Console.WriteLine("Equilateral");
            }
            else if (A != B && B != C && A!= C)
            {
                Console.WriteLine("Scalene");
            }
            else
            {
                Console.WriteLine("Isosceles");
            }
        }
    }
}